test/e2e/cron_test.go

```diff
@@ -360,13 +357,7 @@ spec:
 }
 
 func wfInformerListOptionsFunc(options *v1.ListOptions, cronWfName string) {
-	options.FieldSelector = fields.Everything().String()
-	isCronWorkflowChildReq, err := labels.NewRequirement(common.LabelKeyCronWorkflow, selection.Equals, []string{cronWfName})
-	if err != nil {
-		panic(err)
-	}
-	labelSelector := labels.NewSelector().Add(*isCronWorkflowChildReq)
-	options.LabelSelector = labelSelector.String()
+	options.LabelSelector = common.LabelKeyCronWorkflow + "=" + cronWfName
 }
```

Comment:
@simster7 this *seems* to fix it.

Reply:
I'm wrong, it does not fix it

Reply:
maybe you do get two workflows? one scheduled and one by submit? what about creating the cron workflows in a suspended state? @simster7